src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -489,6 +489,9 @@
   product(bool, DoPartialEscapeAnalysis, false, EXPERIMENTAL,               \
           "Perform partial escape analysis")                                \
                                                                             \
+  product(bool, EnablePassiveMaterialization, true, EXPERIMENTAL,           \
+          "Do passive materialization")                                     \
```

Comment:
I'm in favor of just always doing passive materialization. Without it PEA has regressions.